config/puma.rb

```diff
@@ -42,8 +44,12 @@
 # Specifies the `environment` that Puma will run in.
 environment ENV.fetch('RAILS_ENV') { 'development' }
 
-# Specifies the `pidfile` that Puma will use.
-pidfile ENV.fetch('PIDFILE') { 'tmp/pids/server.pid' }
-
 # Allow puma to be restarted by `bin/rails restart` command.
 plugin :tmp_restart
+
+# Run the Solid Queue supervisor inside of Puma for single-server deployments
+plugin :solid_queue if ENV['SOLID_QUEUE_IN_PUMA']
+
```

Comment:
we are not using solid queue

Reply:
good question.. hmm I find it a bit strange to have code related to non-installed gems.. but whatever you prefer I'm ok

Reply:
Hm yeah, let's remove it